docs/migration/3_0.md

```diff
@@ -102,6 +102,24 @@ There can be more than one cart entry with the same product code. So now they ar
 - `increment` - use new `numberOfEntriesBeforeAdd` instead
 - `cartEntry$` - use `activeCartService.getLastEntry(productCode)` instead
 
+### ActiveCartService
+- Constructor removes `AuthService` and adds `UserIdService` as params.
+
+### MerchandisingCarouselComponent
```

Comment:
Actually this component is not in public API and we don't need migrations docs and migrations for it.